AdminServer/setup.py

```diff
@@ -38,6 +39,8 @@
   packages=['appscale',
             'appscale.admin',
             'appscale.admin.instance_manager'],
+  package_data={'': ['*.json']},
```

Comment:
Usually I see the first argument here specifying the package that should include the data. Is the behavior for an empty string documented somewhere?

Reply:
IIRC this means any of the previously defined packages. We only have resources in one package so it would be easy to change. It seemed simpler as is (no need to list packages again) but now you mention it the behaviour is not obvious.